packages/itwinui-css/backstop/tests/typography.html

```diff
@@ -48,24 +51,73 @@ <h1>Typography</h1>
 
     <section id="demo-typography">
 
-      <h1 class="iui-text-headline iui-text-spacing">Headline</h1>
+      <h1 class="iui-text-headline iui-text-spacing">
+        <svg-placeholder
+          class="iui-icon-inline"
+          data-iui-icon-color="default"
+          aria-hidden="true"
+        ></svg-placeholder>
```

Comment:
These were added to test icon `EM` scaling in relation to the text.  Also tests how the icon displays with skeleton text. https://github.com/iTwin/iTwinUI/pull/810/files#r1020480357